### ISSUE TITLE
track play history

### ISSUE THREAD

**Issue by twillis** [twillis/cljspazzer#18]

**Comment by ttrucco** [comment]
You might have to describe this more. Are we already doing this now because you can click the back button? If so, we can close this.

**Comment by twillis** [comment]
i meant something more like last.fm but stored locally

**Comment by ttrucco** [comment]
Gotcha. This should definitely be a thing. Would love to see most popular etc as we've talked about.

**Comment by twillis** [comment]
yep, most popular etc would be able to be generated from this data
